.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+# Configuration for pre-commit (https://pre-commit.com/), a tool to run
+# formatters, linters, and other productivity tools before a commit.
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.3.0
+    hooks:
+      - id: check-ast
+      - id: check-builtin-literals
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: requirements-txt-fixer
+      - id: check-yaml
+        types: [file]
+        files: ^tests/capi2_cores/.+\.core$
+      - id: end-of-file-fixer
+        # Test golden references
+        exclude: ^tests/(.+\.(info|patch)|test_provider/.+\.v)
+      - id: trailing-whitespace
+        # Test golden references
+        exclude: ^tests/(.+\.(info|patch)|test_provider/.+\.v)
```

Comment:
That's not needed to call black, you can remove it.

Reply:
Done.